Scripts/build.swift

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env swift
```

Comment:
This is based on the build script we're using for [AccessibilitySnapshot](https://github.com/cashapp/AccessibilitySnapshot/blob/fb797dda564408b7da23349b25d192339402286e/Scripts/build.swift) and [Stagehand](https://github.com/cashapp/stagehand/blob/0f8d620939ea4cc2648dba128129bb0917c26090/Scripts/build.swift).